internal/allocation/allocation.go

```diff
@@ -28,6 +28,7 @@ type Allocation struct {
 	Protocol            Protocol
 	TurnSocket          net.PacketConn
 	RelaySocket         net.PacketConn
+	Username            string
```

Comment:
Do we need to export the user name here?

Reply:
Good point.  Let me make it private.